### PR TITLE
Feature/Pixel density and Scale.

### LINKED ISSUE #2319: [SDL 0179] Pixel density and Scale
Proposal: [Pixel density and Scale](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0179-pixel-density-and-scale.md)

> This proposal is about adding parameters to `VideoStreamingCapabilities` for pixel density and scale.

Review: https://github.com/smartdevicelink/sdl_evolution/issues/526

### PR DESCRIPTION
**Pull request by IGapchuk**
Fixed #2319.

This PR is **ready** for review.

### Risk
This PR makes **major** API changes.

### Testing Plan
Covered by Unit tests.

### Summary
Add new parameters to VideoStreamingCapability structure:
- diagonalScreenSize - diagonal screen size of the screen;
- pixelPerInch - Pixel per inch (PPI) of the screen;
- scale - Scale value to determine how the app should scale
the captured view.

### Tasks Remaining:
- [x] [Unit test]

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: src/components/application_manager/rpc_plugins/sdl_rpc_plugin/test/commands/hmi/ui_get_capabilities_response_test.cc [hunk 1]
```diff
@@ -277,7 +277,7 @@ TEST_F(UIGetCapabilitiesResponseTest, SetVideoStreamingCapability_SUCCESS) {
   video_streaming_capability[strings::preferred_resolution]
                             [strings::resolution_width] = 800;
   video_streaming_capability[strings::preferred_resolution]
-                            [strings::resolution_height] = 350;
+                            [strings::resolution_height] = 354;
```
Comment (ValeriiMalkov) [inline src/components/application_manager/rpc_plugins/sdl_rpc_plugin/test/commands/hmi/ui_get_capabilities_response_test.cc]: @IGapchuk Why did you change this value from 350->354 ? Does it make sense ?

Comment (IGapchuk) [reply src/components/application_manager/rpc_plugins/sdl_rpc_plugin/test/commands/hmi/ui_get_capabilities_response_test.cc]: @ValeriiMalkov That changes has been did according to proposal example:
Example 1: 8” screen (800x480 pixels)

scale (1x)
viewport of the head unit (800x354 pixels)
viewport of the apps view (800x354 points)
pixel per inch (117 ppi)
points per inch (117 ppi)

**Comment by IGapchuk** [comment]
rebuild required
